aineko/models/deploy_config_schema_internal.py

```diff
@@ -75,6 +75,30 @@ class Environment(BaseModel, extra="forbid"):
     pipelines: List[Union[str, Dict[str, SpecificPipeline]]]
     load_balancers: Optional[Dict[str, List[LoadBalancer]]]
 
+    @validator("load_balancers")
+    def validate_lb_endpoint(  # pylint: disable=no-self-argument
+        cls, value: Optional[Dict[str, List[LoadBalancer]]]
+    ) -> None | Dict[str, List[LoadBalancer]]:
+        """Load balancer endpoints must be valid.
+
+        The following criteria apply:
+            - Endpoints must be 12 characters or fewer (total 64).
```

Comment:
```suggestion
            - Endpoints must be 12 characters or fewer.
```
Should we include the total amount of characters? This might make it confusing.

Reply:
Added to cloud docs.